server/pxf-jdbc/src/main/java/org/greenplum/pxf/plugins/jdbc/JdbcResolver.java

```diff
@@ -141,6 +128,12 @@ public class JdbcResolver extends JdbcBasePlugin implements Resolver {
 
     private static final Logger LOG = LoggerFactory.getLogger(JdbcResolver.class);
 
+    private boolean logWarnForDateWideRange = false;
+    private static final String dateWideRangeWarnMsg = "Please consider turning on DateWideRange for better performance.";
+    private final String dateWideRangelogMsg = isDateWideRange 

```

Comment:
```suggestion
    private final String dateWideRangeLogMsg = isDateWideRange 
```
this can also be an array of constants (with 0/1 index determined by boolean conversion)